docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -605,7 +608,7 @@ kubectl --context $CLUSTER_SURVIVING get deployments $HELM_RELEASE_NAME-operate
 # camunda-tasklist   0/0     0            0           23m
 ```
 
-For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling.
+For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling. This is a synchronous operation.
```

Comment:
You can also link the api docs above http://localhost:3000/docs/next/self-managed/zeebe-deployment/operations/management-api/#exporting-api

Reply:
linked it where it's executed.